be/src/formats/parquet/file_reader.cpp

```
@@ -474,24 +475,21 @@ Status FileReader::_read_min_max_chunk(const tparquet::RowGroup& row_group, cons
             }
         } else if (!column_meta->__isset.statistics) {
             // statistics not exist in parquet file
-            *exist = false;
-            return Status::OK();
+            return Status::Aborted("No exist statistics");
```

Is no statistics an undefined behavior? If not, `Status::Aborted` seems unappropriate.

---

in caller we just ignore this error, because statistics just helpe running better, if there is no statistics or the type is not supported, we just run ignore the error status.

---

the abort just means the statistics deal process, not the reader, the reader will ignore the error status.